Sources/XcodeGraph/Models/Target.swift

```diff
@@ -227,22 +227,26 @@ public struct Target: Equatable, Hashable, Comparable, Codable, Sendable {
              .macro,
              .dynamicLibrary,
              .staticLibrary,
-             .staticFramework,
              .xpc:
             return false
+
+        case .staticFramework:
+            return containsResources
```

Comment:
Shouldn't we just return true? `supportsResources` and `containsResources` are two different things.

Reply:
You've suggested on tuist/tuist#7006 to make `.staticFramework` embeddable only when it contains resources.
The easiest way to do so in the `tuist`'s codebase is to consider `.staticFramework` as a framework that supports resources only when it really contains it.
Do you have doubts about this approach?

Reply:
Yes, we should still embed `.staticFramework` only if it contains resources. But this variable should always return true for `.staticFramework`, otherwise, we're changing the meaning of this variable for saving ourselves a couple of lines.

I'd check for `containsResources` in the place where we make the decision whether `.staticFramework` should be embedded or not.